server/handlers/tunnel.go

```diff
@@ -156,8 +154,8 @@ func Tunnel(w http.ResponseWriter, r *http.Request) {
 	fmt.Println(r.URL)    // (http://localhost:5000/api/v1 ) > /api/v1
 	fmt.Println("Ravi Adds Path: ", r.URL.Path)
 
-	// backendURL := fmt.Sprintf("http://localhost:8000%s", r.URL)
-	backendURL := fmt.Sprintf(os.Getenv("VITE_BACKEND")+"%s", r.URL)
+	backendURL := fmt.Sprintf("http://localhost:8000%s", r.URL)
```

Comment:
I think we should dynamically specify this

Reply:
Great point. 
Thankfully a super easy fix.
In fact, makes it simpler overall.

Reply:
Complete with a quick change to `server/handlers/tunnel.go` and `server/.env.dev `